src/content/en/updates/2018/09/inside-browser-part1.md

```diff
@@ -0,0 +1,298 @@
+suggests, it was first developed to handle graphics. This is why in the context of graphics 
+"using GPU" or "GPU-backed" is associated with fast rendering and smooth interaction. 
+In recent years, with GPU-accelerated computing, more and more computation is becoming possible on 
+GPU alone.
+
+<div class="clearfix"></div>
+
+When you start an application on your computer or phone, the CPU and GPU are the ones powering 
+the application. Usually, applications run on the CPU and GPU using mechanisms provided by the 
+Operating System.
+
+<figure>
+  <img src="../../images/inside-browser/hw-os-app.png" alt="Hadware, OS, Application">
+  </a>
+  <figcaption>
+   Figure 3: Three layers of computer architecture. Machine Hardware at the bottom, Operating 
+   System in the middle, and Application on top.
+</i>
+  </figcaption>
+</figure>
+
+## Executing program on Process and Thread
+
+<figure class="attempt-right">
+  <img src="../../images/inside-browser/process-thread.png" alt="process and threads">
+  <figcaption>
+   Figure 4: Process as a bounding box, threads as abstract fish swimming inside of a process
+  </figcaption>
+</figure>
+
+Another concept to grasp before diving into browser architecture is Process and Thread. 
+A process can be described as an application’s executing program. A thread is the one that lives 
+inside of process and executes any part of its process program. 
+
+When you start an application, a process is created. The program might create thread(s) to help it 
+do work, but that's optional. The Operating System gives the process a "slab" of memory to work 
+with and all application state is kept in that private memory space. When you close the 
+application, the process also goes away and the Operating System frees up the memory.
+<div class="clearfix"></div>
+
+<figure>
+  <img src="../../images/inside-browser/memory.svg" alt="process and memory">
+  <figcaption>
+   Figure 5: Diagram of a process using memory space and storing application data
+  </figcaption>
+</figure>
+
+A process can ask the Operating System to spin up another process to run different tasks. When this 
+happens, different parts of the memory are allocated for the new process. If two processes need to 
+talk, they can do so by using **I**nter **P**rocess **C**ommunication (**IPC**). Many applications 
+are designed to work this way so that if a worker process get unresponsive, it can be restarted 
+without stopping other processes which are running different parts of the application.
+
+<figure>
+  <img src="../../images/inside-browser/workerprocess.svg" alt="worker process and IPC">
+  <figcaption>
+   Figure 6: Diagram of separate processes communicating over IPC
+  </figcaption>
+</figure>
+
+
+## Browser Architecture {: #browser-architecture }
+
+So how is a web browser built using processes and threads? Well, it could be one process with many 
+different threads or many different processes with a few threads communicating over IPC. 
+<figure>
```

Comment:
The DevSite MD processor can be pick, to be safe, always put a blank line between text and html elements.